### ISSUE TITLE
Change maplayer custom properties 

### ISSUE THREAD

**Issue by enricofer** [3liz/QgisLayerBoardPlugin#9]
Setting layer custom properties http://qgis.org/api/classQgsMapLayer.html#a7bf94cfa08abc37b75cc53dc921bb15e
it's possible to change special layer behaviours like "embedded layers" or for testing or development porpouse.
Regards

**Comment by mdouchin** [comment]
Problem is you can have many custom properties, with many different names, and some layers can have some properties set and others none in the same project. 

The only way would be to add a new tab in the right panel and populate a form based on the custom properties of a single layer.  Some layers could have 3 , some others none. An Apply button could let the user play with the values

**Comment by enricofer** [comment]
You are right, dealing with custom properties complicates user experience, perhaps resetting embedded layers property  could be enough.
